layers/meta-tegrademo/scripts/setup-udev-rules

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
```

Comment:
**Minor:**
Could you please use `#!/bin/sh` instead to make the script compatible with the others shell interpreters like zsh ?

Reply:
done!